e2e/tests/node_not_ready/node_power_off.robot

```diff
@@ -0,0 +1,510 @@
+*** Settings ***
+Documentation       Test the Longhorn resillence if cluster node powering off
+
+Resource            ../../keywords/common.resource
+Resource            ../../keywords/engine.resource
+Resource            ../../keywords/replica.resource
+Resource            ../../keywords/node.resource
+Resource            ../../keywords/volume.resource
+
+Suite Setup         set_test_suite_environment
+Test Setup          set_test_environment    ${TEST NAME}
+Test Teardown       Cleanup resource and resume state
+
+
+*** Variables ***
+${Gi}=                  2**30
+${volume_size_gb}=      1
+${sleep_interval}=      300
+${volume_type}=         RWO
+
+
+*** Test Cases ***
+Node power off with replica-VA
+    [Documentation]    volume with replica on attached node and powering off the volume attached node
+    [Tags]    nodepoweringoff    3replicas    attachreplicasamenode    abnormalreplicanode    abnormalattachednode
```

Comment:
Where are these tags coming from?

Reply:
These are just the labels I set up with my own cognition at the beginning. I'll remove the tags and add them back later after discussing them.